src/views/components/input/tag/InputTag.js

```diff
@@ -35,6 +35,14 @@ class InputTag extends Component {
 
   state = { preSelected: [] }
 
+  shouldComponentUpdate( nextProps ) {
+    // console.warn( 'input tag should component update' );
+    if ( nextProps.items !== this.props.items )
+      return true;
+
+    return false;
+  }
+
```

Comment:
can you remove this block, it is giving me issues with the tag input on native. see if it works without it. i think the issue is that without it, for some reason, the item code is used instead of the name. to solve this, use the prop renderTag for the component, and inside that layout use the value to look up the matching BE and get its name. look at fourdegrees for an example